### ISSUE TITLE
Calendar not pulling new events from Google Calendar

### ISSUE THREAD

**Issue by AbhinavMadahar** [elementary/calendar#360]
Calendar synchronises events from Google Calendar when I first connected it, but it stops pulling updates after that. This is on Elementary OS 5.0

**Comment by AbhinavMadahar** [comment]
I fixed this by installing GNOME Evolution.

**Comment by kr428** [comment]
Tried this workaround on my system; however, "stock" Evolution (without all GNOME Online Accounts) can't connect to Google Apps for Enterprises (with a "custom" domain instead of @google.com). More difficult than I thought. :|

**Comment by kr428** [comment]
Interesting side effect however (not sure this belongs here): I found and installed a package called pantheon-online-accounts-plugin-google which adds "Google" to the options in "Internet accounts" available in elementary control panel; generally this *does* redirect me to Google web login but, in any cases (personal, professional GMail) lets me end up with "your application is not verified yet for Google Sign In". Could this be a related problem?

**Comment by tvdeyen** [comment]
For me the calendar is not loading new events created on other devices from any server. Neither from Google nor from an iCal server. They do appear on other devices if I create them in the elementary Calendar, though, but not the other way round.

**Comment by chances** [comment]
What of this issue? I still cannot add a calendar as "less secure" sign-in strategies are blocked by Google.

**Comment by lfvalerio** [comment]
> What of this issue? I still cannot add a calendar as "less secure" sign-in strategies are blocked by Google.

This seems to be a different issue that, as of now, you need to manually allow "less secure apps" on you [Google Account settings](https://myaccount.google.com/lesssecureapps). I opened a new issue about that because this option is getting removed by Google soon.
